formatter/html.go

```diff
@@ -0,0 +1,77 @@
+package formatter
```

Comment:
I noted below about the interface thing. You can do that here if you want to future proof, but if you're only ever going to be generating html then I think this is fine.